lib/user.cpp

```diff
@@ -52,17 +49,7 @@ class User::Private
 
         qreal makeHueF()
         {
-            Q_ASSERT(!userId.isEmpty());
-            QByteArray hash = QCryptographicHash::hash(userId.toUtf8(),
-                                                       QCryptographicHash::Sha1);
-            QDataStream dataStream(qToLittleEndian(hash).left(2));
-            dataStream.setByteOrder(QDataStream::LittleEndian);
-            quint16 hashValue;
-            dataStream >> hashValue;
-            const auto hueF =
-                    qreal(hashValue)/std::numeric_limits<quint16>::max();
-            Q_ASSERT((0 <= hueF) && (hueF <= 1));
-            return hueF;
+            return stringToHueF(userId);
```

Comment:
`makeHueF()` can be inlined out now. Just call `stringToHueF()` from the constructor.